bin/yaml/libraries.yaml

```diff
@@ -720,10 +720,10 @@ libraries:
       - 12.0.1
       - 13.0.0
       - 13.0.1
-      - 14.0.0
-      - 15.0.0
-      - 16.0.0
-      - 17.0.1
+      - 14.0.6
```

Comment:
Please leave all the prior versions, so please add these patch releases and leave the `x.0.0` versions. We always support all versions forever, we don't delete old versions so that folks trying to see the difference between patch releases (e.g. bug fixes) can compare between them.